index.ts

```diff
@@ -208,6 +208,8 @@ namespace logger {
       req.transaction.end();
     });
 
+    res.setHeader('X-FC-Transaction', req.transaction.id);
```

Comment:
Does having this here have the implication of starting to send the response immediately after the request is received? Just worried this might case side effects somewhere